retrieval/programmatic-access/tag_querying.rst

```diff
@@ -244,12 +239,41 @@ table, to see what they apply to.
 How are the Tags Created?
 -------------------------
 
-The tags are typically assigned by automatic processes analysing the user supplied metadata around an object. 
-
-For example, the identification of “marine” sample records is systematically assessed by a combination of geo-coordinates and taxonomic evidence. We can further qualify such identification by a level of confidence which is dictated by a combination of the evidence available on the record to support said assertion. 
+The tags are typically assigned by automatic processes analysing the user supplied metadata around an object.
 
 This is an evolving and continuously improving process, where the algorithms and the rule-sets used for classification can be updated as new insights are obtained and thus results in the assigned tags being regularly refreshed. The flexibility of this system allows for new classifications to be easily created allowing the definition of new, high-level contextual groupings for ENA data making the process of discovery more intuitive for certain user communities.
 
+^^^^^^^^
+pathogen
+^^^^^^^^
+The pathogen tags are identified based on tax IDs. The different types of pathogens have a maintained list of tax ids, all records associated with the tax ID or a tax ID in that lineage get the pathogen tag.
```

Comment:
based on NCBI taxonomy IDs.

Reply:
We maintain curated lists of tax ids for the different types of pathogens, and records associated with the such a tax ID or a descendant of that tax ID will get the "pathogen" tag.